pandaserver/dataservice/adder_atlas_plugin.py

```diff
@@ -237,324 +235,321 @@ def update_outputs(self):
+                    if file.lfn in self.extra_info["nevents"]:
+                        file_attrs["events"] = self.extra_info["nevents"][file.lfn]
+                    elif self.extra_info["nevents"] != {}:
+                        file_attrs["events"] = None
+                    # if file.jediTaskID not in [0,None,'NULL']:
+                    #    fileAttrs['task_id'] = file.jediTaskID
+                    file_attrs["panda_id"] = file.PandaID
+                    if campaign not in ["", None]:
+                        file_attrs["campaign"] = campaign
+
+                # extract OS files
+                has_normal_url = True
+                if file.lfn in self.extra_info["endpoint"] and self.extra_info["endpoint"][file.lfn] != []:
+                    for pilot_end_point in self.extra_info["endpoint"][file.lfn]:
+                        # pilot uploaded to endpoint consistently with original job definition
+                        if pilot_end_point in dataset_destination_map[file_destination_dispatch_block]:
+                            has_normal_url = True
+                        else:
+                            # alternative stage-out
+                            alt_staged_files.add(file.lfn)
+                            map_for_alt_stage_out.setdefault(pilot_end_point, {})
+                            map_for_alt_stage_out[pilot_end_point].setdefault(file_destination_dispatch_block, [])
+                            map_for_alt_stage_out[pilot_end_point][file_destination_dispatch_block].append(file_attrs)
+
+                if has_normal_url:
+                    # add file to be added to dataset
+                    id_map[file_destination_dispatch_block].append(file_attrs)
+                    # add file to be added to zip
+                    if not is_zip_file and zip_file_name:
+                        if "files" not in zip_files[zip_file_name]:
+                            zip_files[zip_file_name]["files"] = []
+                        zip_files[zip_file_name]["files"].append(file_attrs)
+                    if is_zip_file and not self.add_to_top_only:
+                        # copy file attribute for zip file registration
+                        for tmp_file_attr_name, tmp_file_attr_val in file_attrs.items():
+                            zip_files[file.lfn][tmp_file_attr_name] = tmp_file_attr_val
+                        zip_files[file.lfn]["scope"] = file.scope
+                        zip_files[file.lfn]["rse"] = dataset_destination_map[file_destination_dispatch_block]
+
+                # for subscription
                 if (
-                    self.job.jobStatus == "finished"
-                    and EventServiceUtils.isEventServiceJob(self.job)
-                    and not EventServiceUtils.isJobCloningJob(self.job)
-                    and file.type != "log"
+                    self.job.prodSourceLabel in ["managed", "test", "software", "user", "rucio_test"] + JobUtils.list_ptest_prod_sources
```

Comment:
I can't think of any other prodsourcelabel, not clear what's the point of this line. Was it to exclude the 'ddm' prodsourcelabel in the past?

('software' doesn't exist anymore)

Reply:
The line excludes jobs with prodsourcelabel=panda, but the next line essentially excludes them. I think OK to remove the condition with prodsourcelabel.